src/types/ErrorMessages.ts

```diff
@@ -0,0 +1,21 @@
+export enum ErrorMessages {
+  NONE,
+  LOAD,
+  TITLE,
+  ADD,
+  DELETE,
+  DELETE_COMPLETED,
+  UPDATE,
+  UPDATE_ALL,
+}
+
+// export enum ErrorMessages {
+//   NONE = '',
+//   LOAD = 'Unable to load todos',
+//   TITLE = 'Title can\'t be empty',
+//   ADD = 'Unable to add the todo',
+//   DELETE = 'Unable to delete the todo',
+//   DELETE_COMPLETED = 'Unable to delete completed todos',
+//   UPDATE = 'Unable to update a todo',
+//   UPDATE_ALL = 'Unable to update status of todos',
+// }
```

Comment:
as for me, this looks much better

Reply:
@alexandra-protyanova  As for me, I agree with you. But after  @Anastasiia-Khudych  comment, I read a lot of information about enum, and noticed that she was right, usually, enum is used for a short list of values, not for storing all the text of error. So I decided to leave the second version.

Reply:
ok, in that case, let's delete the commented code